vision/alexnet_cifar10/README.md

```diff
@@ -0,0 +1,19 @@
+# AlexNet
+
+<p align="center">
+  <img src="docs/AlexNet.png" />
+</p>
+
+The neural network, which has 60 million parameters and 650,000 neurons, consists of five convolutional layers, some of which are followed by max-pooling layers, and three fully-connected layers with a final 1000-way softmax. To make train- ing faster, we used non-saturating neurons and a very efficient GPU implemen- tation of the convolution operation. To reduce overfitting in the fully-connected layers we employed a recently-developed regularization method called “dropout” that proved to be very effective. We also entered a variant of this model in the ILSVRC-2012 competition and achieved a winning top-5 test error rate of 15.3%, compared to 26.2% achieved by the second-best entry.
```

Comment:
This appears to be the literal abstract from a paper. Linking to sources is fine, but we'd want to have an original description that is tailored for the zoo.

Reply:
Oh, I thought the original abstract was enough. I'll fix it soon, thank you!

Reply:
Fixed 😁